eZ/Bundle/EzPublishRestBundle/Resources/config/input_parsers.yml

```diff
@@ -650,6 +660,60 @@ services:
         tags:
             - { name: ezpublish_rest.input.parser, mediaType: application/vnd.ez.api.internal.sortclause.SectionName }
 
+    ezpublish_rest.input.parser.internal.facetbuilder.content_type:
+        parent: ezpublish_rest.input.parser
+        class: "%ezpublish_rest.input.parser.internal.facetbuilder.content_type.class%"
+        tags:
+            - { name: ezpublish_rest.input.parser, mediaType: application/vnd.ez.api.internal.facetbuilder.ContentType }
+
+    ezpublish_rest.input.parser.internal.facetbuilder.criterion:
+        parent: ezpublish_rest.input.parser
+        class: '%ezpublish_rest.input.parser.internal.facetbuilder.criterion.class%'
+        tags:
+            - { name: ezpublish_rest.input.parser, mediaType: application/vnd.ez.api.internal.facetbuilder.Criterion }
+
+    ezpublish_rest.input.parser.internal.facetbuilder.date_range:
+        parent: ezpublish_rest.input.parser
+        class: '%ezpublish_rest.input.parser.internal.facetbuilder.date_range.class%'
+        tags:
+            - { name: ezpublish_rest.input.parser, mediaType: application/vnd.ez.api.internal.facetbuilder.DateRange }
+
+    ezpublish_rest.input.parser.internal.facetbuilder.field:
+        parent: ezpublish_rest.input.parser
+        class: '%ezpublish_rest.input.parser.internal.facetbuilder.field.class%'
+        tags:
+            - { name: ezpublish_rest.input.parser, mediaType: application/vnd.ez.api.internal.facetbuilder.Field }
+
+    ezpublish_rest.input.parser.internal.facetbuilder.field_range:
+        parent: ezpublish_rest.input.parser
+        class: '%ezpublish_rest.input.parser.internal.facetbuilder.field_range.class%'
+        tags:
+            - { name: ezpublish_rest.input.parser, mediaType: application/vnd.ez.api.internal.facetbuilder.FieldRange }
+
+    ezpublish_rest.input.parser.internal.facetbuilder.location:
+        parent: ezpublish_rest.input.parser
+        class: '%ezpublish_rest.input.parser.internal.facetbuilder.location.class%'
+        tags:
+            - { name: ezpublish_rest.input.parser, mediaType: application/vnd.ez.api.internal.facetbuilder.Location }
+
+    ezpublish_rest.input.parser.internal.facetbuilder.section:
+        parent: ezpublish_rest.input.parser
+        class: '%ezpublish_rest.input.parser.internal.facetbuilder.section.class%'
+        tags:
+            - { name: ezpublish_rest.input.parser, mediaType: application/vnd.ez.api.internal.facetbuilder.Section }
+
+    ezpublish_rest.input.parser.internal.facetbuilder.term:
+        parent: ezpublish_rest.input.parser
+        class: '%ezpublish_rest.input.parser.internal.facetbuilder.term.class%'
+        tags:
+            - { name: ezpublish_rest.input.parser, mediaType: application/vnd.ez.api.internal.facetbuilder.Term }
+
+    ezpublish_rest.input.parser.internal.facetbuilder.user:
+        parent: ezpublish_rest.input.parser
+        class: '%ezpublish_rest.input.parser.internal.facetbuilder.user.class%'
```

Comment:
> @alongosz had a second look here?

Just waiting for @Nattfarinn to replace remaining single quotes with double quotes around parameters, because now we have a mix in this file ;)